core/sys/linux/types.odin

```diff
@@ -120,6 +120,25 @@ when ODIN_ARCH == .amd64 {
 		_:         [3]uint,
 	}
 } else when ODIN_ARCH == .arm64 {
+	_Arch_Stat :: struct {
+		dev:        Dev,
+		ino:        Inode,
+		mode:       Mode,
+		nlink:      u32,
+		uid:        Uid,
+		gid:        Gid,
+		rdev:       Dev,
+		_:          u64,
+		size:       int,
+		blksize:    i32,
+		_:          i32,
+		blocks:     int,
+		atime:      Time_Spec,
+		mtime:      Time_Spec,
+		ctime:      Time_Spec,
+		_:          [2]u32,
+	}
+} else when ODIN_ARCH == .riscv64 {
```

Comment:
Are you use this is correct...? You added these lines above where you check for arm64, not for riscv

Reply:
Looks like arm64 and riscv64 have the same struct for stat actually, don't know where I got the wrong idea from, good spot